docs/layouts/page/playground.html

```diff
@@ -1,12 +1,33 @@
 {{ define "header" }}
 {{ partial "header.html" . }}
+
 {{ end }}
 
+
 {{ define "main" }}
+<style>
```

Comment:
This file is overwritten by a build process that creates the actual file used so we can't put this here. But, it can be put into the script for the build process.